backend/pkg/util/notifications.go

```diff
@@ -0,0 +1,45 @@
+package util
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/SherClockHolmes/webpush-go"
+	"github.com/kioku-project/kioku/pkg/model"
+	"go-micro.dev/v4/logger"
+)
+
+func SendNotification(subscription *model.PushSubscription, notification *model.PushNotification) {
```

Comment:
let the error handling be done by the caller

```suggestion
func SendNotification(subscription *model.PushSubscription, notification *model.PushNotification) error {
```

Reply:
Implemented in 2a3ba4b